ringpop.go

```diff
@@ -89,8 +89,7 @@ type Ringpop struct {
 		sync.RWMutex
 	}
 
-	logger log.Logger
-	log    log.Logger
+	moduleLogger *util.ModuleLogger
```

Comment:
Any reason why we would name this `moduleLogger` over something simpler like `logger` or just `log`?